alert-policies/apache-traffic-server/FreeCacheRAM.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Free Cache RAM Percent
```

Comment:
```suggestion
name: Free RAM Cache Percent
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.